src/app/shared/pipes/timestamp.pipe.ts

```diff
@@ -0,0 +1,18 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'timestamp'
+})
+export class TimestampPipe implements PipeTransform {
+  public transform(timestampInSeconds: number): string {
+    const seconds: number = timestampInSeconds % 60
+    let minutes: number = Math.floor(timestampInSeconds / 60)
+    let hours: number = 0
+    if (minutes >= 60) {
+      hours = Math.floor(minutes / 60)
+      minutes = minutes % 60
+    }
+
+    return `${hours > 0 ? `${hours}:` : ''}${minutes < 10 ? `0${minutes}` : minutes}:${seconds < 10 ? `0${seconds}` : seconds}`
```

Comment:
This line can be simplified by doing the following.

```typescript
public transform(timestampInSeconds: number): string {
   ...
   return `${hours > 0 ? `${hours}:` : ''}${this.zeroPadStart(minutes)}:${this.zeroPadStart(seconds)}`
}

private zeroPadStart(nr: number): string {
   return nr.toString().padStart(2, '0')
 }
```

Reply:
Jeg har opdateret klassen til at bruge `.padStart`, men også selve strukturen for formatteringen.

Reply:
What are you saying? :D

Reply:
Sorry :grin: I have updated the class to use `.padStart`, but also how the formatting is applied.